packages/sirius-web/frontend/sirius-web-application/src/views/project-browser/list-projects-area/useProjects.types.ts

```diff
@@ -17,11 +17,26 @@ export interface UseProjectsValue {
   refreshProjects: () => void;
 }
 
+export interface ProjectFilter {
```

Comment:
Why the need for two interfaces for the same concept? Just use `GQLProjectFilter`

Reply:
Done